tests/CSR.php

```diff
@@ -7,14 +7,47 @@ class CSR extends Base
     public function setUp(): void
     {
         parent::setUp();
+
+        Client::$port = 3001;
+        $this->awaitPortOpen();
+        Client::$port = 3000;
     }
 
-    public function testHomepage(): void
+    // TODO: Move secret checks to base.php
```

Comment:
Reminder 2

Reply:
Aware of this, making myself note. I will address in separate PR. Want to keep TODO to be sure its not forogtten